src/test/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/StocksServiceImplTests.java

```diff
@@ -0,0 +1,221 @@
+import com.google.api.client.http.HttpResponseException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.handler.smallfull.SmallFullResourceHandler;
+import uk.gov.companieshouse.api.handler.smallfull.stocks.StocksResourceHandler;
+import uk.gov.companieshouse.api.handler.smallfull.stocks.request.StocksGet;
+import uk.gov.companieshouse.api.model.accounts.smallfull.stocks.StocksApi;
+import uk.gov.companieshouse.web.accounts.api.ApiClientService;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.model.smallfull.BalanceSheet;
+import uk.gov.companieshouse.web.accounts.model.smallfull.BalanceSheetHeadings;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.stocks.PaymentsOnAccount;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.stocks.Stocks;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.stocks.StocksNote;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.stocks.Total;
+import uk.gov.companieshouse.web.accounts.service.smallfull.BalanceSheetService;
+import uk.gov.companieshouse.web.accounts.service.smallfull.StocksService;
+import uk.gov.companieshouse.web.accounts.transformer.smallfull.StocksTransformer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class StocksServiceImplTests {
+
+    @Mock
+    private ApiClientService mockApiClientService;
+
+    @Mock
+    private ApiClient mockApiClient;
+
+    @Mock
+    private SmallFullResourceHandler mockSmallFullResourceHandler;
+
+    @Mock
+    private StocksResourceHandler mockStocksResourceHandler;
+
+    @Mock
+    private StocksGet mockStocksGet;
+
+    @Mock
+    private StocksTransformer mockStocksTransformer;
+
+    @Mock
+    private BalanceSheetService mockBalanceSheetService;
+
+    @Mock
+    private BalanceSheet mockBalanceSheet;
+
+    @InjectMocks
+    private StocksService stocksService = new StocksServiceImpl();
+
+    private static final String TRANSACTION_ID = "transactionId";
+
+    private static final String COMPANY_ACCOUNTS_ID = "companyAccountsId";
+
+    private static final String COMPANY_NUMBER = "companyNumber";
+
+    private static final String BASE_SMALL_FULL_URI = "/transactions/" + TRANSACTION_ID +
```

Comment:
Again, is this required? We could just have a fully qualified `STOCKS_URI` and avoid the concatenation

Reply:
done